docs/src/main/tut/generate-sources-from-idl/compendium.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
**requirement**: i think you need to add the link to the menu: `docs/src/main/resources/microsite/data/menu.yml`